src/stash.c

```diff
@@ -83,9 +83,9 @@
 #include <stdlib.h> /* only for atoi() */
 
 
+/* GTK3 removed ComboBoxEntry, but we need a value to differentiate combo box with and
+ * without entries, and it must not collide with other GTypes */
```

Comment:
I'm open to a rewording, but I think it's important to have some kind of a comment here, because the reason for this is not so obvious I think.